passport.js

```diff
@@ -0,0 +1,54 @@
+const passport = require("passport"),
+  LocalStrategy = require("passport-local").Strategy,
+  Models = require("./models.js"),
+  passportJWT = require("passport-jwt");
+
+let Users = Models.User,
+  JWTStrategy = passportJWT.Strategy,
+  ExtractJWT = passportJWT.ExtractJwt;
+
+passport.use(
+  new LocalStrategy(
+    {
+      usernameField: "username",
+      passwordField: "password",
+    },
+    (username, password, callback) => {
+      console.log(username + " " + password);
+      Users.findOne({ username: username }, (error, user) => {
+        if (error) {
+          console.log(error);
```

Comment:
Let us not push console.log debugging to `git/version` control. You can always delete them after testing. Remember we used `morgan` please read on how you can work with it in-depth.